<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xdiffr</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBoxPath1">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonBrowse1">
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDiffTextEdit" name="textEditDiff1" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBoxPath2">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonBrowse2">
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDiffTextEdit" name="textEditDiff2" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCompare"/>
   <addaction name="actionFolders"/>
   <addaction name="actionPreviousChange"/>
   <addaction name="actionNextChange"/>
   <addaction name="actionSettings"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCompare">
   <property name="icon">
    <iconset resource="xdiffr.qrc">
     <normaloff>:/resources/images/res/toolbar-compare.png</normaloff>:/resources/images/res/toolbar-compare.png</iconset>
   </property>
   <property name="text">
    <string>Compare</string>
   </property>
   <property name="toolTip">
    <string>Compare/re-compare</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionFolders">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="xdiffr.qrc">
     <normaloff>:/resources/images/res/toolbar-folderdiff.png</normaloff>:/resources/images/res/toolbar-folderdiff.png</iconset>
   </property>
   <property name="text">
    <string>Folders</string>
   </property>
   <property name="toolTip">
    <string>Compare Folders</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
  <action name="actionPreviousChange">
   <property name="icon">
    <iconset resource="xdiffr.qrc">
     <normaloff>:/resources/images/res/toolbar-prevdiff.png</normaloff>:/resources/images/res/toolbar-prevdiff.png</iconset>
   </property>
   <property name="text">
    <string>PreviousChange</string>
   </property>
   <property name="toolTip">
    <string>Previous Change</string>
   </property>
   <property name="shortcut">
    <string>Alt+Up</string>
   </property>
  </action>
  <action name="actionNextChange">
   <property name="icon">
    <iconset resource="xdiffr.qrc">
     <normaloff>:/resources/images/res/toolbar-nextdiff.png</normaloff>:/resources/images/res/toolbar-nextdiff.png</iconset>
   </property>
   <property name="text">
    <string>NextChange</string>
   </property>
   <property name="toolTip">
    <string>Next Change</string>
   </property>
   <property name="shortcut">
    <string>Alt+Down</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="xdiffr.qrc">
     <normaloff>:/resources/images/res/toolbar-about.png</normaloff>:/resources/images/res/toolbar-about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="xdiffr.qrc">
     <normaloff>:/resources/images/res/toolbar-settings.png</normaloff>:/resources/images/res/toolbar-settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDiffTextEdit</class>
   <extends>QWidget</extends>
   <header>qdifftextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="xdiffr.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionCompare</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionCompare_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFolders</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionFolders_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPreviousChange</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onClickPreviousChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNextChange</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onClickNextChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onClickAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onClickSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionCompare_triggered()</slot>
  <slot>actionFolders_triggered()</slot>
  <slot>onClickPreviousChange()</slot>
  <slot>onClickNextChange()</slot>
  <slot>onClickAbout()</slot>
  <slot>onClickSettings()</slot>
 </slots>
</ui>
